src/lib/components/parameters.svelte

```diff
@@ -45,6 +45,12 @@
     );
   };
 
+  const engineeringMode = () => {
```

Comment:
Could this be renamed to be a verb? I'm thinking something consistent with `enterConfigurationMode` on L20:

```suggestion
  const enterEngineeringMode = () => {
```